assets/queries/openAPI/2.0/security_definitions_using_password_flow/query.rego

```diff
@@ -0,0 +1,19 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) == "2.0"
+
+	sec_def := doc.securityDefinitions[key]
+	sec_def.flow == "password"
```

Comment:
I think it should be `secDef`. Also, I think we should check `secDef.type == "oauth2"`

Reply:
the name 'sec_def' is to follow REGO naming convention.
I think it is not necessary to check if 'type' is "oauth" since the flow field Is only used for type "oauth", meaning that it won't exist in other type making the query fail on line 10

Reply:
I agree with you about `type`, but for variables, we use camel case style in rego files